rust_src/src/textprop.rs

```diff
@@ -19,4 +22,13 @@ pub fn get_char_property(position: EmacsInt, prop: LispObject, object: LispObjec
     unsafe { get_char_property_and_overlay(position.into(), prop, object, ptr::null_mut()) }
 }
 
+/// Return the value of POSITION's property PROP, in OBJECT.
+/// OBJECT should be a buffer or a string; if omitted or nil, it defaults
+/// to the current buffer.
+/// If POSITION is at the end of OBJECT, the value is nil.
+#[lisp_fn(min = "2")]
+pub fn get_text_property(position: EmacsInt, prop: LispObject, object: LispObject) -> LispObject {
+    unsafe { textget(Ftext_properties_at(position.into(), object), prop) }
```

Comment:
Do you want to try porting `text-properties-at` as well?

Reply:
Yes! I think I'll do it in another pull request, though, since it's likely to take me longer than this one (I'm a newbie :)).